src/app/core.py

```diff
@@ -0,0 +1,469 @@
+"""This file contains the core logic used in the SPDX Online Tools' APP and API"""
+
+import jpype
+import os
+
+from django.utils.datastructures import MultiValueDictKeyError
+from django.core.files.storage import FileSystemStorage
+from django.conf import settings
+
+from json import dumps, loads
+from time import time
+from traceback import format_exc
+from urllib.parse import urljoin
+from app.views import formatToContentType, getFileFormat
```

Comment:
Move these functions to utils.py, views.py should just be a wrapper over helper functions.

Reply:
Alright, I will move these functions to utils.py file